<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>110</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>290</width>
    <height>110</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Edit Planet Name</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.ico</normaloff>img/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(47, 47, 47);
color:rgb(200, 200, 200)
}
QTabWidget::pane {
border: 1px solid rgb(128, 128, 128);
}
QTabBar::tab 
{
    background: #48555E;
    color: white;   
	padding:5px;
    border: 1px solid #000;
}

QTabBar::tab:selected, 
QTabBar::tab:hover 
{
    border-top-color: #1D2A32;
    border-color: #40494E;
    color: black;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #C1D8E8, stop: 1 #F0F5F8); 
}
QLineEdit, QDateEdit, QComboBox{
	background-color: rgb(85,85,85);
	border-color: rgb(0, 0, 0);
	border-width : 1px;
	border-style:inset;
	padding: 5px;
	border-radius:3px;
}
QPushButton{
	background:rgb(102, 102, 102);
	color:rgb(255, 255, 255);
	border-radius:3px;
	padding-top:5px;
	padding-bottom:5px;
	padding-right:10px;
	padding-left:10px;
	border-color: rgb(0, 0, 0);
	border-width : 1px;
	border-style:inset;
}
QPushButton:hover{
	background-color:rgb(128, 128, 128);
}
QPushButton:pressed{
	background-color:#A4A4A4;
	color:#000
}
QLabel{
	border:0px;
}</string>
  </property>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>78</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPlanetName">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>10</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Planet Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>13</y>
     <width>9</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>73</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCoordinate">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>43</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>46</y>
     <width>9</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>46</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coordinate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
